frontend/src/components/pages/AddProduct.js

```diff
@@ -28,33 +27,24 @@ const AddProduct = () => {
         ProductDataService.create(product)
             .then(() => {
 
-                setSubmitted(true);
+                alert("Produdo cadastrado com sucesso!!!")
```

Comment:
Tem um errinho aqui:

```suggestion
                alert("Produto cadastrado com sucesso!!!")
```